src/sentry/models/team.py

```diff
@@ -160,6 +160,7 @@ class Team(Model, SnowflakeIdMixin):
     idp_provisioned = models.BooleanField(default=False)
     date_added = models.DateTimeField(default=timezone.now, null=True)
     org_role = models.CharField(max_length=32, null=True)
+    through_project_creation = models.BooleanField(default=False)
```

Comment:
is this a permanent field? or will we drop it after the experiment is done?

Reply:
hmm I presume we would drop if after the experiment, although if it ends up successful and we want to reimplement it properly we could re-use this field